webapp/channels/src/utils/embed.tsx

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+/**
+ * Returns true if the user is running Mattermost in an embedded view
+ */
+export function isEmbedded(): boolean {
+    return document.cookie.indexOf('MMEMBED=1') !== -1;
```

Comment:
Are we not doing the `window.self !== window.parent` check from https://github.com/mattermost/mattermost/pull/30810 ?

Reply:
Totally right, this is a leftover from a cherry pick. Fixed.